torchvision/transforms/functional_tensor.py

```diff
@@ -3,12 +3,19 @@
 from torch.jit.annotations import List, BroadcastingList2
 
 
-def _is_tensor_a_torch_image(input):
-    return input.ndim >= 2
+@torch.jit.export
```

Comment:
I'm not sure `@torch.jit.export` is needed?

Reply:
Yes, you are right about that, I'll remove it